data/README.md

```diff
@@ -45,6 +45,7 @@
 | plain-dict-uncompressed-checksum.parquet         | uncompressed and dictionary-encoded INT32 and STRING columns in format v1 with a matching CRC |
 | rle-dict-uncompressed-corrupt-checksum.parquet   | uncompressed and dictionary-encoded INT32 and STRING columns in format v2 with a mismatching CRC |
 | large_string_map.brotli.parquet       | MAP(STRING, INT32) with a string column chunk of more than 2GB. See [note](#large-string-map) below |
+| concatenated_gzip_members.parquet     | 513 int64 numbers compressed using 2 gzip members |
```

Comment:
```suggestion
| concatenated_gzip_members.parquet     | 513 UINT64 numbers compressed using 2 gzip members |
```

Reply:
why you choosed [U]INT64 ?
parquet-mr shows INT64:

          type      encodings count     avg size   nulls   min / max
long_col  INT64     G RB_     513       2.86 B             "1" / "513"

Reply:
The schema shows that its logical type is unsigned.
```
message root {
  optional int64 long_col (INTEGER(64,false));
}
```

Reply:
yes, I missed the logical type. thanks